internal/pkg/pluginmanager/downloader_test.go

```diff
@@ -1,29 +1,55 @@
 package pluginmanager
 
 import (
+	"errors"
+	"fmt"
+	"log"
 	"os"
 	"path/filepath"
 	"testing"
 
+	"github.com/go-resty/resty/v2"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestDownload(t *testing.T) {
-	os.Remove(filepath.Join(".", "argocdapp_0.0.1-rc1.so"))
+func mockPlugGetter(reqClient *resty.Client, url, plugName string) error {
+	return nil
+}
+
+func mockPlugNotFoundGetter(reqClient *resty.Client, url, plugName string) error {
+	return errors.New("Plug Not Exist")
+}
 
+func TestDownloadSuccess(t *testing.T) {
+	tmpDir := t.TempDir()
+	plugName := "argocdapp_0.0.1-rc1.so"
+	version := "0.0.1-ut-do-not-delete"
```

Comment:
Please consider refactoring `plugName` and `version` as constants.

Reply:
yeah，constant should be a better choice